bin/audit.py

```diff
@@ -0,0 +1,152 @@
+# from gfactoryXmlToDB import *
+from collections import defaultdict
+import yaml
+import requests
+import sys
+import urllib.request
+try:
+    import xml.etree.cElementTree as ET
+except ImportError:
+    import xml.etree.ElementTree as ET
+
+
+treeDump = False
+
+
+def getGfactoryData(gfactoryDB, xml):
+    response = urllib.request.urlopen(xml)
+    gfactoryPage = response.read()
+    root = ET.fromstring(gfactoryPage)
+
+    for entries in root.findall('entries'):
+        for entry in entries.findall('entry'):
+            for attrs in entry.findall('attrs'):
+                for attr in attrs.findall('attr'):
```

Comment:
```suggestion
            for attrs in entry.findall('attrs/attr'):
```

Similarly, I think this should work.